server/methods/alexaApiCall.js

```diff
@@ -0,0 +1,38 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+import { check } from 'meteor/check';
+
+import { settings } from '../../app/settings';
+
+Meteor.methods({
+	registerAlexaUser(serverurl, servername) {
+		check(serverurl, String);
+		check(servername, String);
+
+		const userid = Meteor.userId();
```

Comment:
Please use camel case for variable names
```
	registerAlexaUser(serverUrl, serverName) {
		check(serverUrl, String);
		check(serverName, String);

		const userId = Meteor.userId();
```

Reply:
this variables are actually being used for MongoDB model, so I have not used Camel Case else it will not work.

Reply:
Where are you using it on mongodb? I couldn't find it.

Reply:
here => https://github.com/RocketChat/vui-multiservers-proxy/blob/master/models/Register.js

Reply:
I see, you can always change your model there or map the values when calling the API here, but in this project, we use Camel Case as standard.

Reply:
can I leave this variables as it is or else I will have to change models in Proxy server code as well as in Alexa code also ??

Reply:
As I said, you can call the API using the names you want, but not keep the variables in lower case, you can map them.

Reply:
ok changed the variable names to camelCase 👍